framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -131,6 +131,9 @@ public static AnnotatedTypeMirror createType(
     /** Actual type wrapped with this AnnotatedTypeMirror. */
     protected final TypeMirror underlyingType;
 
+    /** The hash code for {@code underlyingType.toString()}. Lazily initialized. */
```

Comment:
I would document that the hash code for this is not cached because ATM is mutable, but the underlying type never changes and is immutable, so its hash code does not change.

Reply:
Nit: Java spells the method `hashCode` so I would capitalize the C here as well.